extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -610,6 +662,7 @@ export class AmpConsent extends AMP.BaseElement {
         return false;
       }
       // Prompt
+      this.consentStateChangedViaPromptUI_ = true;
```

Comment:
this boolean should be set to true when the localStorage is updated via UI.

Reply:
moved to `handleAction()`, where user action prompts `consentStateManager_.updateConsentInstance(accept/reject/dismiss)`.